modules/web/src/app/settings/admin/defaults/template.html

```diff
@@ -297,6 +297,56 @@
           <km-spinner-with-confirmation [isSaved]="isEqual(settings.defaultNodeCount, apiSettings.defaultNodeCount)"></km-spinner-with-confirmation>
         </div>
 
+        <div fxLayout="row">
+          <div fxFlex="16%"
+               fxLayoutAlign=" center"
+               class="entry-label">
+            <span>Allowed Operating System</span>
+            <div class="km-icon-info km-pointer"
+                 matTooltip="Allowed operating system shows if the operating system is allowed to be use with machinedeployment,
+                  and it is only applicable for the supported operating systems for a cloud provider.">
+            </div>
+          </div>
+
+          <mat-button-toggle-group group="operatingSystemGroup"
+                                   [(ngModel)]="allowedOperatingSystems"
+                                   (change)="onOperatingSystemChange($event.value)"
+                                   multiple
+                                   fxLayout="row wrap"
+                                   kmValueChangedIndicator>
+            <mat-button-toggle [value]="OperatingSystem.Ubuntu"
+                               [disabled]="isLastEnabledOS(OperatingSystem.Ubuntu)">
```

Comment:
Can we put this `[disabled]` on the `mat-button-toggle-group` instead?

Reply:
that will disable all the `mat-button-toggle` , what im trying to do here is to prevent the admin from disable all the os so if there is only one os is enabled then the admin can't disable it